Update windows documentation (#18703)
netdata/netdata: packaging/windows/WINDOWS_INSTALLER.md

    - [GPLv3 License](/LICENSE): Read the GNU General Public License v3, which governs the Netdata software.
3. **Destination**:  Choose the installation directory. By default, Netdata installs in `C:\Program Files\Netdata`.
4. **Installation**: The installer will copy the necessary files to the chosen directory.

> 5. **Connecting**: In order to [connect](/src/claim/README.md) your Agent to your Netdata Cloud Space you need to provide the following:
>     - **Claim Token**: This is the Claim Token that will be used to connect the Agent to your Space.
>     - **Room IDs**: These are the Room IDs where your Agent will be added. Multiple IDs are separated by a comma `,`.
>     - **Proxy address**: Enter the address of a proxy server if required for communication with Netdata Cloud.

Reviewer: ```suggestion
5. **Connect to Netdata Cloud**: To [connect](/src/claim/README.md) your Agent to your Netdata Cloud Space you need to provide the following:
    - **Claim Token**: The Claim Token that securely authenticates and links your Agent to your Space.
    - **Room IDs**: A comma-separated list of Room IDs where you want to add your Agent.
    - **Proxy address**: The address of a proxy server, if one is required for communication with Netdata Cloud.
```

    - **Insecure connection**: By default, Netdata verifies the server's certificate. Enabling this option bypasses verification (use only if necessary).
    - **Open Terminal**: Select this option to launch the `MSYS2` terminal after installation completes.
6. **Finish**: The installation process is complete!

## Silent Mode (Command line)

This section provides instructions for installing Netdata in silent mode, which is ideal for automated deployments.

> **Info**
>

> > In order to have no interaction with the install process, make sure to run the installation file as an administrator. Otherwise the Windows admin prompt popup will appear

Reviewer: ```suggestion
> Run the installer as admin to avoid the Windows prompt.
```

Straight to the point

```

Replace `<YOUR_TOKEN>` and `<YOUR_ROOM>` with your actual Netdata Cloud Space claim token and room ID, respectively.

> **Note**
>

> > If you are running the installation for the first time, MSYS2 will also initialize.

Reviewer: > will also initialize

what does it mean?

Author: This was weird, maybe @thiagoftsm actually knows what happens in this case, a MSYS2 window popped up and I think it had nothing to do with Netdata, probably it was trying to initialize everything. After reinstalling, it didn't display the messages it did at first

Reviewer: I mean, what is "MSYS2 initialization"?

Author: MSYS2 is a command line environment like Windows CMD when I fist ran the installer it started and produced an output that I didn't capture, but it looked like some sort of first-time run

Reviewer: From the user's point of view, "MSYS2 is also initialized" is completely unclear. I suggest deleting this line or rewording.

Author: Given I don't know how it works, let's remove it and leave it to the user to assume that the installation invokes the MSYS2 window.

## Uninstalling

> You can uninstall Netdata via running the `uninstall.exe`, located under `<YOUR_INSTALL_LOCATION>\Netdata`.

Reviewer: ```suggestion
To uninstall Netdata, run the `uninstall.exe` file in your Netdata installation directory, typically `<YOUR_INSTALL_LOCATION>\Netdata`.
```
